runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/ParserErrors/TokenMismatch2.stg

```diff
@@ -10,7 +10,7 @@ Grammar ::= [
 
 Input() ::= "( ~FORCE_ERROR~ "
 
-Rule() ::= "a"
```

Comment:
:thought_balloon: I'm really curious how this passed the initial testing. It seems obviously wrong.